test/smoke_test/smoke_test.py

```diff
@@ -26,3 +30,6 @@ def s3_test():
     options = parser.parse_args()
     if options.s3:
         s3_test()
+    
+    if torchdata.__version__ >= "0.8.0": 
```

Comment:
Please throw exception here of torchdata.__version != 0.8.0 
We would need to route it later via input parameter. However this can be a followup PR